Userland/Games/Solitaire/main.cpp

```diff
@@ -150,6 +150,24 @@ int main(int argc, char** argv)
         statusbar.set_text(2, "Timer starts after your first move");
     };
 
+    window->on_close_request = [&]() {
+        auto game_in_progress = timer->is_active();
+        if (game_in_progress) {
+            auto result = GUI::MessageBox::show(window,
+                "A game is still in progress, are you sure you would like to quit?",
+                "Game in progress",
+                GUI::MessageBox::Type::Warning,
+                GUI::MessageBox::InputType::YesNoCancel);
```

Comment:
What is the significance of pressing "Cancel" here?

Reply:
None, this should be YesNo. Will amend :)